BootloaderCommonPkg/Library/BootloaderCommonLib/BootloaderCommonLib.c

```diff
@@ -783,3 +785,62 @@ GetHashToExtend (
 
   return Status;
 }
+

```

Comment:
I think this should be in some of the security related library.  Not common lib here.

Reply:
Done. Moved this API to TpmLib.